common/metrics/otel_metrics_handler.go

```diff
@@ -108,9 +114,12 @@ func (omp *otelMetricsHandler) Timer(timer string) TimerIface {
 	})
 }
 
-// Histogram obtains a histogram for the given name and MetricOptions.
-func (omp *otelMetricsHandler) Histogram(histogram string, unit MetricUnit) HistogramIface {
-	c, err := omp.provider.GetMeter().Int64Histogram(histogram, instrument.WithUnit(otelunit.Unit(unit)))
+// Histogram obtains a histogram for the given name and Option list.
+func (omp *otelMetricsHandler) Histogram(histogram string, unit MetricUnit, opts ...Option) HistogramIface {
+	c, err := omp.provider.GetMeter().Int64Histogram(histogram, buildOtelOptions[instrument.Int64Option](
```

Comment:
what is the reason not just use instrument.Option?

Reply:
Because `instrument` is an otel package, and we have other metrics.Handler implementations.